K2Bridge.Tests.End2End/KustoClientTests.cs

```diff
@@ -91,6 +91,9 @@ public async Task FieldCaps_WithAllTypes_ReturnsTypesConvertedToESTypes()
         {
             var fieldCaps = await K2Client().FieldCaps(typesIndexFullName);
             var expected = JObject.Parse(@"{
+              ""indices"": [
+                ""types_index""
```

Comment:
can  usetypes IndexFullName

(need to take substring)

Reply:
👍🏻